scripts/core-triage/project.py

```diff
@@ -0,0 +1,195 @@
+# imports
+import os
+import requests
+
+# TODO: improve
+from graphql_queries import *
+
+# TODO: what is error handling?
+
+# constants # TODO: move to a config file (or something) <= copilot suggested this last part
+ORG = "dbt-labs"
+PROJECT_NAME = "Core triage"
+PROJECT_NUM = 22
+# list of Core teams
+CORE_TEAMS = ["core"]
+# read newline separated list of projects from file
+REPOS = [
+    "dbt-core",
+    "dbt-redshift",
+    "dbt-bigquery",
+    "dbt-snowflake",
+    "dbt-rpc",
+    #"dbt-server", # internal, causes failure
+    "dbt-spark",
+    #"dbt-starter-project", # not important (?)
+    "dbt-utils",
+    "homebrew-dbt",
+]
+# issues added by label individually; lots of duplication here
+ISSUE_LABELS = [
+    "triage",
+    "bug",
+    "good_first_issue",
+    "help_wanted",
+    "spike",
+    "python_models",
+    "Team:Language",
+    "Team:Execution",
+    "Team:Adapters",
+]
+# prs added by label individually
+PR_LABELS = ["ready_for_review"]
+# GitHub TOKEN environment variable name
+TOKEN_VAR = "GH_TOKEN"
+# Number of items per query -- max is 100 TODO: paging?
+NUM_ITEMS = 100
+
+# API stuff -- not uppercase because...
+headers = {"Authorization": f"token {os.environ.get(TOKEN_VAR)}"}
+gh_api_url = "https://api.github.com"
+gh_graphql_url = f"{gh_api_url}/graphql"
+
+# functions
+def process_request(url: str, headers: dict, json: dict = None) -> dict:
+    """
+    Process a request to the GitHub API (GET or POST, based on json).
+    ---
+    Inputs: url, headers, json (optional)
+    Outputs: response (dict)
+    """
+    if json:
+        r = requests.post(url, headers=headers, json=json)
+    else:
+        r = requests.get(url, headers=headers)
+
+    print(r.status_code)
+    print(r.text)
+
+    return r.json()
+
+
+def get_core_members(teams: list[str]) -> list:
+    """
+    Get a list of Core members' GitHub logins (usernames).
+    ---
+    Inputs: teams (list of str)
+    Outputs: core_members (list of str)
+    """
+    # initialize empty list
+    core_members = []
+
+    # for each core team, add its members
+    for team in teams:
```

Comment:
Do we see adding more teams at some point?  Language, Execution and Adapters are all under Core right?  Seems unnecessary to loop through a list with a length of 1.  Maybe `teams` could just be a string?

Reply:
maybe? I was trying to future-proof it but removing, easy enough to make work on multiple teams later